<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Connect_Arduino">
            <property name="text">
             <string>Conectar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Disconnect_Arduino">
            <property name="text">
             <string>Desconectar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextBrowser" name="Display">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="Clean_Display">
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="Motor_1">
            <property name="text">
             <string>Motor 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Motor_2">
            <property name="text">
             <string>Motor 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Both_Motor">
            <property name="text">
             <string>Ambos</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="Select_Velocity_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Seleccionar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QSpinBox" name="Velocity_1">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>Velocidad: </string>
            </property>
            <property name="minimum">
             <number>4000</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>9000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="Cycle_1">
            <property name="text">
             <string>Ciclo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Test_1">
            <property name="text">
             <string>Prueba</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="Open_1">
            <property name="text">
             <string>Abre</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="Close_1">
            <property name="text">
             <string>Cierra</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="Advance0_1">
            <property name="text">
             <string>Avanzar (0)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="Advance1_1">
            <property name="text">
             <string>Avanzar (1)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QSpinBox" name="Velocity_2">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="prefix">
             <string>Velocidad: </string>
            </property>
            <property name="minimum">
             <number>4000</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>9000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Test_2">
            <property name="text">
             <string>Prueba</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="Cycle_2">
            <property name="text">
             <string>Ciclo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="Select_Velocity_2">
            <property name="text">
             <string>Seleccionar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="Open_2">
            <property name="text">
             <string>Abre</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="Close_2">
            <property name="text">
             <string>Cierra</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="Advance1_2">
            <property name="text">
             <string>Avanzar (1)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="Advance0_2">
            <property name="text">
             <string>Avanzar (0)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="Quit_App"/>
   </widget>
   <widget class="QMenu" name="menuAjustes">
    <property name="title">
     <string>Ajustes</string>
    </property>
    <addaction name="Serial_Port"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de </string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAjustes"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Serial_Port">
   <property name="text">
    <string>Puerto </string>
   </property>
  </action>
  <action name="Quit_App">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
